libsolidity/ast/Types.cpp

```diff
@@ -600,7 +600,7 @@ FixedPointType::FixedPointType(unsigned _totalBits, unsigned _fractionalDigits,
 {
 	solAssert(
 		8 <= m_totalBits && m_totalBits <= 256 && m_totalBits % 8 == 0 &&
-		0 <= m_fractionalDigits && m_fractionalDigits <= 80,
+		0 < m_fractionalDigits && m_fractionalDigits <= 80,
```

Comment:
Hm, I'm not sure if we actually force `m_fractionalDigits` to be nonzero somewhere.

Reply:
Should I keep it as an `int`?

Reply:
No, just remove the lower bound check.